src/lib/components/input/stories/config.ts

```diff
@@ -0,0 +1,56 @@
+const sharedConfig = {
+  value: {
+    defaultValue: '',
+    description: 'Current input value.',
+    control: { type: 'text' }
+  },
+  className: {
+    description: 'Wrapper classNames for custom styling',
+    defaultValue: '',
+    control: { type: 'text' }
+  },
+  placeholder: {
+    description: 'Placeholder for DirtySwan Input component',
+    defaultValue: 'Placeholder',
+    control: { type: 'text' }
+  },
+  label: {
+    description: 'Label of the Input component',
+    defaultValue: 'Input Label',
+    control: { type: 'text' }
+  },
+  hideLabel: {
+    description: 'Wether a label should be hidden. This is needed for accessibility purposes and a label should always be provided',
+    defaultValue: false,
+    control: { type: 'boolean' }
+  },
+  prefix: {
+    description: 'A preffix that can be displayed on the left of the input. Specially useful for currencies.',
+    defaultValue: '£',
+    control: { type: 'text' }
+  },
+  id: {
+    description: 'Unique ID identifier of the input. Relevent for accessibility.',
```

Comment:
I think you have a typo here
```suggestion
    description: 'Unique ID identifier of the input. Relevant for accessibility.',
```

Reply:
Thanks for catching all those typos 😅 😅

Reply:
That one works great for me: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker
You wouldn't believe how much stuff it uncovers haha

Reply:
Ohh I'll get this one, I think it's relevent to me 😄